config.go

```diff
@@ -28,9 +29,15 @@ var (
 	allowedRecipients = flag.String("allowed_recipients", "", "Regular expression for valid 'to' email addresses")
 	deniedRecipients  = flag.String("denied_recipients", "", "Regular expression for email addresses for which will never deliver any emails.")
 	// set "" to allow any user (i.e disable users check)
-	allowedUsers = flag.String("allowed_users", "", "Path to file with valid users/passwords")
-	remoteHost   = flag.String("remote_host", "smtp.gmail.com:587", "Outgoing SMTP server")
-	remoteUser   = flag.String("remote_user", "", "Username for authentication on outgoing SMTP server")
+	allowedUsers   = flag.String("allowed_users", "", "Path to file with valid users/passwords")
```

Comment:
we need to add these options in https://github.com/grafana/smtprelay/blob/master/smtprelay.ini as well

Reply:
Good catch! Updated 😄